hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java

```diff
@@ -206,6 +207,18 @@ private static Stream<Arguments> scanTestCases() {
             Named.of("Filter dataSize>2000", Arrays.asList("--filter", "dataSize:greater:2000")),
             Named.of("Expect empty result", null)
         ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter key3 regex", Arrays.asList("--filter", "keyName:regex:^.*3$")),
+            Named.of("Expect key3", Pair.of("key3", "key4"))
+        ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter dataSize>500 using regex", Arrays.asList("--filter", "dataSize:regex:^5.*$")),
```

Comment:
nit : ^5.*$ means any data size starting with 5 
```suggestion
            Named.of("Filter keys whose dataSize digits start with 5 using regex", Arrays.asList("--filter", "dataSize:regex:^5.*$")),
```

Reply:
Thanks for the review, I have updated the test now